concepticondata/conceptlists/Steinthal-1875-1565.tsv

```diff
@@ -0,0 +1,1566 @@
+Steinthal-1875-1565-560	560	Kleider reinigen	Kleider reinigen	to clean clothes	3434	WASH (CLOTHES)
+Steinthal-1875-1565-561	561	waschen	waschen	to wash	1453	WASH
+Steinthal-1875-1565-562	562	trocknen	trocknen	to dry	1786	DRY UP
+Steinthal-1875-1565-563	563	schmutzig	schmutzig	dirty, unclean	1230	DIRTY
+Steinthal-1875-1565-564	564	rein	rein	clean	704	CLEAN
+Steinthal-1875-1565-565	565	nass	nass	wet	1726	WET
+Steinthal-1875-1565-566	566	Schmuck	Schmuck	adornment	917	ORNAMENT
+Steinthal-1875-1565-567	567	einzelne Gegenstände zum Schmuck, wie Ohrring, Armring u.s.w.	Ohrring	earring	770	EARRING
+Steinthal-1875-1565-568	568	einzelne Gegenstände zum Schmuck, wie Ohrring, Armring u.s.w.	Armring	armlet	3149	ARMLET
+Steinthal-1875-1565-569	569	Haus	Haus	house	1252	HOUSE
+Steinthal-1875-1565-570	570	Hütte	Hütte	hut	1678	HUT
+Steinthal-1875-1565-571	571	Zelt aufschlagen, abbrechen	Zelt	tent	1316	TENT
+Steinthal-1875-1565-572	572	Zelt aufschlagen, abbrechen	Zelt aufschlagen	to erect a tent		
+Steinthal-1875-1565-573	573	Zelt aufschlagen, abbrechen	Zelt abbrechen	to strike a tent		
+Steinthal-1875-1565-574	574	Zeltpflock	Zeltpflock	tent stake		
+Steinthal-1875-1565-575	575	Küche (Heerd)	Küche	kitchen	1365	KITCHEN
+Steinthal-1875-1565-576	576	Küche (Heerd)	Heerd	hearth, stove	859	STOVE
+Steinthal-1875-1565-577	577	Schlafstätte	Schlafstätte	sleeping place		
+Steinthal-1875-1565-578	578	Bett	Bett	bed	1663	BED
+Steinthal-1875-1565-579	579	Thür	Thür	door	1567	DOOR
+Steinthal-1875-1565-580	580	Thorweg	Thorweg	gateway	2940	GATE
+Steinthal-1875-1565-581	581	Schlüssel (Riegel)	Schlüssel	key	1472	KEY
+Steinthal-1875-1565-582	582	Schlüssel (Riegel)	Riegel	bolt	1012	LATCH
+Steinthal-1875-1565-583	583	Schloss	Schloss	lock	1596	LOCK
+Steinthal-1875-1565-584	584	verschliessen	verschliessen	to lock		
+Steinthal-1875-1565-585	585	zumachen	zumachen	to close	1046	SHUT
+Steinthal-1875-1565-586	586	öffnen	öffnen	to open, to unlock		
+Steinthal-1875-1565-587	587	Fenster	Fenster	window	1375	WINDOW
+Steinthal-1875-1565-588	588	Rauchloch	Rauchloch	smoke hole		
+Steinthal-1875-1565-589	589	Treppe	Treppe	stairs	34	STAIRS
+Steinthal-1875-1565-590	590	oberes Geschoss	oberes Geschoss	upper floor		
+Steinthal-1875-1565-591	591	Dach	Dach	roof	769	ROOF
+Steinthal-1875-1565-592	592	Wand	Wand	wall	933	WALL (OF HOUSE)
+Steinthal-1875-1565-593	593	Balken	Balken	bar		
+Steinthal-1875-1565-594	594	Thürpfosten	Thürpfosten	door jamb		
+Steinthal-1875-1565-595	595	Zeltpflock	Zeltpflock	tent stake		
+Steinthal-1875-1565-596	596	bauen	bauen	to build	1840	BUILD
+Steinthal-1875-1565-597	597	zimmern	zimmern	to do carpentry		
+Steinthal-1875-1565-598	598	Matte flechten	Matte flechten	to weave a mat		
+Steinthal-1875-1565-599	599	Tisch	Tisch	table	1775	TABLE
+Steinthal-1875-1565-600	600	Stuhl	Stuhl	chair	1476	CHAIR
+Steinthal-1875-1565-601	601	Bank	Bank	bench	2584	BENCH
+Steinthal-1875-1565-602	602	Beil	Beil	axe	677	AXE
```

Comment:
I thought since he has the concept twice in the list, there must be a difference, e.g. one is the big thing for felling trees (axe) and the other is the more hand-sized tool for splitting firewood (hatchet), but that's a lot of interpretation. They can both be axe.